inference_segmentation.py

```diff
@@ -354,10 +360,6 @@ def main(params: Union[DictConfig, dict]) -> None:
     num_classes = num_classes + 1 if num_classes > 1 else num_classes  # multiclass account for background
     num_bands = len(bands_requested)
 
-    working_folder = state_dict.parent.joinpath(f'inference_{num_bands}bands')
-    logging.info("\nThe state dict path directory used '{}'".format(working_folder))
-    Path.mkdir(working_folder, parents=True, exist_ok=True)
-    logging.info(f'\nInferences will be saved to: {working_folder}\n\n')
```

Comment:
The output folder will change to a more simple hierarchy. The destination folder will become the user's choice via the "inferences.root_dir" parameter.

Reply:
Can you confirm if evaluate_segmentation works with the new hierarchy?

Reply:
Good point. I will check.